src/stedi/cdk.clj

```diff
@@ -1,11 +1,57 @@
 (ns stedi.cdk
   (:refer-clojure :exclude [require import])
-  (:require [clojure.data.json :as json]
+  (:require [clojure.java.browse :as browse]
+            [clojure.data.json :as json]
+            [clojure.string :as string]
             [clojure.java.io :as io]
             [stedi.cdk.impl :as impl]
             [stedi.cdk.import :as import]
             [stedi.cdk.jsii.client :as client]))
 
+(def ^:private docs-prefix
+  "https://docs.aws.amazon.com/cdk/api/latest/docs")
```

Comment:
How hard would it be to get a reference to the version of the modules that are bundled?  Some modules are marked alpha and are under active development; given the rate of change on CDK, it might be nice to help users avoid frustration by pointing users to docs that are newer and describe different interfaces to what we include.